<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Oyun</class>
 <widget class="QMainWindow" name="Oyun">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Oyun</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#Oyun{
	background-image: url(:/new/images/background1.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="hayalet">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>500</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="slash">
    <property name="geometry">
     <rect>
      <x>940</x>
      <y>340</y>
      <width>171</width>
      <height>171</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="koordinat">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>321</width>
      <height>111</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>koordss</string>
    </property>
   </widget>
   <widget class="QLabel" name="meyve_1">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="meyve_2">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>310</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="meyve_3">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>370</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="meyve_4">
    <property name="geometry">
     <rect>
      <x>810</x>
      <y>380</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="meyve_5">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>410</y>
      <width>101</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="fullscreen">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>20</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Tam Ekran</string>
    </property>
   </widget>
   <widget class="QPushButton" name="normal">
    <property name="geometry">
     <rect>
      <x>1820</x>
      <y>80</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Pencereli</string>
    </property>
   </widget>
   <widget class="QLabel" name="can">
    <property name="geometry">
     <rect>
      <x>124</x>
      <y>120</y>
      <width>161</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x </string>
    </property>
   </widget>
   <widget class="QLabel" name="can_hayalet">
    <property name="geometry">
     <rect>
      <x>45</x>
      <y>107</y>
      <width>100</width>
      <height>110</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/images/can.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="skor">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>30</y>
      <width>591</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>48</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Skor : 0</string>
    </property>
   </widget>
   <widget class="QLabel" name="kaybettin">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>260</y>
      <width>1151</width>
      <height>501</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>72</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dash">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>490</y>
      <width>841</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dash_hakki_acik">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>830</y>
      <width>461</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dash_hakki_kapali">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>830</y>
      <width>481</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>28</pointsize>
      <weight>75</weight>
      <italic>true</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">image: url(:/new/images/dash_kapali.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="hasar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="kontrol">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1920</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>hayalet</zorder>
   <zorder>meyve_1</zorder>
   <zorder>meyve_2</zorder>
   <zorder>meyve_3</zorder>
   <zorder>meyve_4</zorder>
   <zorder>meyve_5</zorder>
   <zorder>slash</zorder>
   <zorder>dash</zorder>
   <zorder>hasar</zorder>
   <zorder>skor</zorder>
   <zorder>koordinat</zorder>
   <zorder>kaybettin</zorder>
   <zorder>dash_hakki_kapali</zorder>
   <zorder>dash_hakki_acik</zorder>
   <zorder>can_hayalet</zorder>
   <zorder>can</zorder>
   <zorder>kontrol</zorder>
   <zorder>normal</zorder>
   <zorder>fullscreen</zorder>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>fullscreen</sender>
   <signal>clicked()</signal>
   <receiver>Oyun</receiver>
   <slot>showFullScreen()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>424</x>
     <y>68</y>
    </hint>
    <hint type="destinationlabel">
     <x>468</x>
     <y>224</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>normal</sender>
   <signal>clicked()</signal>
   <receiver>Oyun</receiver>
   <slot>showNormal()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>508</x>
     <y>55</y>
    </hint>
    <hint type="destinationlabel">
     <x>563</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
